src/nrnpython/rxd.cpp

```diff
@@ -341,7 +341,14 @@ void apply_node_flux(int n, long* index, double* scale, PyObject** source, doubl
             {
                 src = (PyHocObject*)source[i];
                 /*TODO: check it is a reference */
-                states[j] +=  dt * *(src->u.px_) / scale[i];
+                if(src->type_ == PYHOC_REF_TO_NUMBER)
```

Comment:
If you do the change above, and if this file has the right `#include`, this would become a comparison to `PyHoc::HocRefNum`.